src/components/ElevationProfile.js

```diff
@@ -0,0 +1,71 @@
+import { useMemo } from 'react';
+import styled from "styled-components";
+import * as turf from "@turf/turf";
+import CanvasJSReact from "../assets/lib/canvasjs.react.js";
+const { CanvasJSChart } = CanvasJSReact;
+
+const StyledElevationProfile = styled.header`
+  // border: 2px solid orange;
```

Comment:
if you want to comment out a line in a styled component you have to use `/*` instead of `//` I believe

Reply:
The /* syntax is for commenting out a block of text (with */ to close that comment). The // syntax is just for commenting out a single line.

Reply:
My bad, my editor does not recognize the `//` syntax for commenting out within styled-components. You're right, it works!